distribution/docs/src/main/resources/content/_devComponents/custom-transformers-input.adoc

```diff
@@ -15,13 +15,13 @@ ${branding} supports the creation of custom <<_types_of_transformers,input trans
 `public Metacard transform(InputStream input, String id) throws IOException, CatalogTransformerException`
 . Import the ${branding} interface packages to the bundle manifest (in addition to any other required packages). +
 `Import-Package: ddf.catalog,ddf.catalog.transform`
-. Create an OSGi descriptor file to communicate with the OSGi Service Registry (described in the <<_osgi_basics,Working with OSGi>> section). Export the service to the OSGi Registry and declare service properties.
+. Create an OSGi descriptor file to communicate with the OSGi Service Registry (described in the <<_osgi_basics,OSGi Basics>> section). Export the service to the OSGi Registry and declare service properties.
```

Comment:
❓ Should we be putting the `{developing-prefix}` here like we do in other places? Like `{developing-prefix}osgi_basics`.

Reply:
yes. fixed.